css/main.less

```diff
@@ -58,12 +58,16 @@ p, a, li {
   color: @wet-asphalt;
 }
 
+.landingPage-title {
+    margin-top: 0em;
+}
+
 h1, h2, h3, h4, h5, h6 {
-  font-family:'Lato', sans-serif;
-  color: @primary-brand;
-  font-weight: 700;
-  margin-bottom: 0.5em;
-  margin-top: 2em
+    font-family: 'Lato', sans-serif;
```

Comment:
According to [§2.2.1](https://google.github.io/styleguide/htmlcssguide.html#Indentation) of the Google HTML/CSS Style Guide (as adopted in [process](/oss-generic/process)), 2 spaces should be used as indentation.